src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SingleSpaceSeparatorCheck.java

```diff
@@ -67,6 +67,14 @@
  * <pre>
  * &lt;module name=&quot;SingleSpaceSeparator&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * int foo()   { &#47;&#47;Violation, 3 whitespaces
```

Comment:
`&#47;&#47;` no need to escape, you can use slashes

Please correct here and below